src/modules/course/dto/create-course.dto.ts

```diff
@@ -79,13 +79,6 @@ export class CreateCourseDto {
   @IsOptional()
   readonly sharable_link: string;
 
-  /**
-   * The Mentor of the course
-   * @example ['John Doe']
-   */
-  @IsArray()
-  mentor: [];
-
```

Comment:
Please don't remove Mentors from create-course dto since it is necessary to add a mentor at creation

Reply:
ok